types/wordpress__components/autocomplete/index.d.ts

```diff
@@ -114,6 +115,8 @@ declare namespace Autocomplete {
         onChange?(value: Value): void;
         onReplace?(value: Value): void;
         record?: Value;
+        instanceId?: string;
```

Comment:
I believe this is an internal property you can't provide. It's provided by the `withInstanceId` that wraps `Autocomplete`.

https://github.com/WordPress/gutenberg/blob/68cb40e67f62729b29aa63df9794d55875748837/packages/components/src/autocomplete/index.js#L538-L540

Otherwise, this seems to better [match usage](https://github.com/WordPress/gutenberg/blob/aa0abb915e843e2d636d4242aa249e504284abbc/packages/block-editor/src/components/rich-text/index.js#L624-L630).

Reply:
Oh gosh, good catch, thanks!